### ISSUE TITLE
Support for Java 17, the current Java with LTS

### ISSUE THREAD

**Issue by mh182** [docToolchain/docToolchain#1071]
Most Linux distribution provide the last Java LTS as default Java runtime. As of today (2023-03) this is Java 17. 
Supporting Java 17 would the need to download a Java version just for `docToolchain`.

What impediments do we have to upgrade to Java 17?

**Comment by rdmueller** [comment]
the biggest impediment is the upgrade of the asciidoctor gradle plugin. This plugin follows a new approach which makes it hard to switch the old code to the new plugin.

**Comment by mh182** [comment]
Next steps: after #1098 is merged into `ng` expand the test matrix to Java 17 and check what errors are popping up.

**Comment by PacoVK** [comment]
Java 17 support is available since Gradle 7.3 [see](https://docs.gradle.org/current/userguide/compatibility.html), however as described in #1104  we currently cannot go further than 7.0.2
